WordPress/gutenberg
# Remove the verb Toggle from the Block Inserter button.

Addresses part of https://github.com/WordPress/gutenberg/issues/61483

## What?

- The verb 'toggle' isn't well translatable in many languages and should not be used.
- WordPress features should be title case, much like 'List View', 'Document Overview', and many other ones.

Note: I'd still like the 'Block Inserter' to be renamed because the current name is too technical and the tool isn't just inseerting blocks any longer. However, this is a first step to make the name better translatable and avoid the verb 'toggle'.

## Why?

- Better usability and localization.
- Consistency

## How?

- Chagees all the strings 'Toggle block inserter` to 'Block Inserter'.

## Testing Instructions




- Edit a post.
- Hover or focus the block inserter button. 
- Observe the tooltip is 'Block Inserter', title case.
- Check the same think in the Site Editor and in the Widgets editor.
- Make sure there are no occurrences of the string 'Toggle block inserter' aanywhere in the codebase.
- Check the tests pass.

### Testing Instructions for Keyboard


## Screenshots or screencast
